### PR TITLE
Add `Copy Download Link` FAQ

### LINKED ISSUE #345: FAQ: What does the `Copy Download Link` button do?
This should cover the following:
- Allows users to get the download link for the project to download the data onto a server using cmds like wget
- Link expires in 7 days - so use with caution. You can get a new link from the portal as many times as you need.
- Download links aren't available for samples, only for projects

Link to this FAQ from the CHANGELOG

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
## Purpose

I am adding an FAQ that explains the Copy Download Link button on the Portal. Per the issue, I am also adding a short CHANGELOG entry.

### Issue addressed

Closes #345 - I am choosing to go directly to `main` here, given the small scope, and we'll follow up with a merge `main` -> `development`.

## Checklist for publishing to `main`

- [x] This branch contains all commits/content ready to be released.
- [ ] The most recent CHANGELOG entry includes a date.
If this release required a CHANGELOG entry, it should be today's date.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
The spellcheck failure can be safely ignored – the placeholder date is what is triggering the failure.

**Review by jashapiro — APPROVED** [review]
This looks fine in content, but do we want it to go to `main` or `development`?

**Review by dvenprasad — APPROVED** [review]
LGTM

🐐 🐐 🐐

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## PLACEHOLDER YYYY.MM.DD
```
Comment (jaclyn-taroni) [inline docs/CHANGELOG.md:15]: We expect this to go in tomorrow:
```suggestion
## 2024.10.01
```

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
...
+
+* You can now use a `Copy Download Link` button to get download links for projects on the ScPCA Portal.
+Please see the {ref}`FAQ <faq:what does the copy download link button do>` for more information.
```
Comment (jaclyn-taroni) [inline docs/CHANGELOG.md:18]:
```suggestion
Please see the {ref}`FAQ <faq:What does the Copy Download Link button do?>` for more information.
```

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,11 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## PLACEHOLDER YYYY.MM.DD
+
+* You can now use a `Copy Download Link` button to get download links for projects on the ScPCA Portal.
+Please see the {ref}`FAQ <faq:What does the Copy Download Link button do?>` for more information.
```
Comment (jaclyn-taroni) [inline docs/CHANGELOG.md:18]:
```suggestion
Please see the {ref}`FAQ <faq:What does the \`Copy Download Link\` button do?>` for more information.
```

**Pull request merged**